coordinator/points_writer.go

```diff
@@ -353,6 +353,12 @@ func (w *PointsWriter) writeToShard(shard *meta.ShardInfo, database, retentionPo
 		return nil
 	}
 
+	// If this is a partial write error, that is also ok.
```

Comment:
@jwilder can you verify this is the correct thing to do here?  Otherwise it was re-trying which has no effect for a partial write.